kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2030,6 +2030,22 @@ object KyuubiConf {
       .intConf
       .createOptional
 
+  val SERVER_LIMIT_CONNECTIONS_CUSTOM_ENABLE: ConfigEntry[Boolean] =
+    buildConf("kyuubi.server.limit.connections.custom.enable")
+      .doc("Whether to enable custom limit. If it is enabled, Kyuubi will limit the" +
```

Comment:
how does custom mean?

Reply:
early，the config is kyuubi.server.limit.connections.userdefine.enable. After, I  thought userdefine is also unthinkable. therefore, using custom representing customized param which can be defiend by Users themselves

Reply:
if you want to put them into upstream, then custom is not suitable.

Reply:
What do you think of a more straightforward title? For example **throttle** or **downgrade**?

`kyuubi.server.throttle.enable`?

Reply:
I have makd some adjust and commited according your suggestion